crates/iota/src/client_commands.rs

```diff
@@ -191,7 +191,7 @@ pub enum IotaClientCommands {
         #[arg(long)]
         tx_bytes: String,
         /// A list of Base64 encoded signatures `flag || signature || pubkey`.
```

Comment:
```suggestion
        /// A list of Base64 encoded signatures `flag || signature || pubkey`, separated by space.
```
not fur if fmt is happy with that

Reply:
https://github.com/iotaledger/iota/pull/7409/commits/c4606d8c62556a78ccc02b9f90b67aace746d656